include/aws/auth/auth.h

```diff
@@ -51,6 +51,7 @@ enum aws_auth_errors {
     AWS_AUTH_CREDENTIALS_PROVIDER_SSO_SOURCE_FAILURE,
     AWS_AUTH_IMDS_CLIENT_SOURCE_FAILURE,
     AWS_AUTH_PROFILE_STS_CREDENTIALS_PROVIDER_CYCLE_FAILURE,
+    AWS_AUTH_CREDENTIALS_PROVIDER_ECS_INVALID_TOKEN_FILE_PATH,
```

Comment:
does sep say it should fail or just be treated as if it was not set?

Reply:
Yes, it says If AWS_CONTAINER_AUTHORIZATION_TOKEN_FILE is set, and the file cannot be opened or read for any reason (doesn’t exist, no permissions, invalid bytes, etc.), implementations must fail to resolve.